packages/react/src/components/toggleButton/ToggleButton.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axe } from 'jest-axe';
+
+import { ToggleButton } from './ToggleButton';
+
+describe('<ToggleButton /> spec', () => {
+  it('renders the component', () => {
+    const { asFragment } = render(<ToggleButton id="toggle-button" value={false} label="Test label" />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should not have basic accessibility issues', async () => {
+    const { container } = render(<ToggleButton id="toggle-button" value={false} label="Test label" />);
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+  });
+});
```

Comment:
Not much functionality to test here, but maybe we could test the toggle status? We could apply the attribute aria-pressed to check pressing the button toggles its state.

Reply:
Yes, I could add that one 👍

Reply:
Added tests